hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -126,6 +126,8 @@ cdef class PureMarketMakingStrategy(StrategyBase):
         self._ask_place_threshold = ask_place_threshold
         self._order_size = order_size
         self._cancel_order_wait_time = cancel_order_wait_time
+        # Add radar relay type exchanges where you can expire orders instead of cancelling them
+        self._radar_relay_type_exchanges = ['RadarRelayMarket', 'BambooRelayMarket']
```

Comment:
Since you're checking for membership here... using a Python set would be more appropriate.